### PR TITLE
📝 Update Deta deployment tutorial for compatibility with Deta Space

### PR DESCRIPTION
**Pull request by sudo-subg**

### CODE REVIEW

**Comment by mikBighne98** [comment]
WIP / draft

**Comment by Ryandaydev** [comment]
Are there any tests associated with this PR?

**Comment by mikBighne98** [comment]
> Are there any tests associated with this PR?
@Ryandaydev, no. Just an update on how to install the FastAPI app on the Deta for compatibility with Space

File: docs/en/docs/deployment/deta.md [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
 
 @app.get("/")
-def read_root():
+def root():
     return {"Hello": "World"}
 
 
 @app.get("/items/{item_id}")
 def read_item(item_id: int):
+		print({"item_id": item_id})
```
Comment (santigandolfo) [inline docs/en/docs/deployment/deta.md:36]:
```suggestion
    print({"item_id": item_id})
```

File: docs/en/docs/deployment/deta.md [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
-def read_root():
-    return {"Hello": "World"}
+async def root():
+    return {"hello": "world"}
 
 
 @app.get("/items/{item_id}")
-def read_item(item_id: int):
+async def read_item(item_id: int):
+    print({"item_id": item_id})
```
Comment (abdelhai) [inline docs/en/docs/deployment/deta.md:36]: would remove

**Review by abdelhai — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
Thanks @mikBighne98! :cake: 

I tweaked the docs a bit, fixed a couple of things, updated a couple others for consistency, and fixed the console outputs, this is ready! :rocket:

**Comment by mikBighne98** [comment]
Thank you for reviewing and merging the PR, @tiangolo :slightly_smiling_face:

**Pull request merged**